### PR TITLE
CLN: Fix F901 and E306 mixups

### PR DESCRIPTION
**Pull request by jbrockmendel**
Ref #5325

### CODE REVIEW

**Comment by jbrockmendel** [comment]
@ChadFulton raising NotImplemented is egregious enough that it should actually be fixed.

**Comment by ChadFulton** [comment]
Thanks @jbrockmendel

File: statsmodels/stats/_lilliefors.py [hunk 1]
```diff
@@ -166,6 +166,7 @@ def get_lilliefors_table(dist='norm'):
 
 
         # also build a table for larger sample sizes
+
```
Comment (thequackdaddy) [inline statsmodels/stats/_lilliefors.py]: @jbrockmendel I think we actually should've removed lines here. flake8 can be confusing/finicky

Comment (thequackdaddy) [reply statsmodels/stats/_lilliefors.py]: I made a quick fix is #5335 ... let's see if I know what I'm doing.

**Pull request merged**